### PR TITLE
Add previewFileMimeTypes to defaultconfig.go

### PR DESCRIPTION
**Pull request by JanAckermann**
## Description
Enhancement: Add previewFileMimeTypes to web default config

We've added previewFileMimeTypes to the web default config, so web can determine which preview types are supported
by the backend.

## Related Issue

- Fixes <issue_link>

## Motivation and Context
Part of: https://github.com/owncloud/web/pull/7474

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: services/web/pkg/config/defaults/defaultconfig.go [hunk 1]
```diff
@@ -49,7 +49,8 @@ func DefaultConfig() *config.Config {
 				},
 				Apps: []string{"files", "search", "preview", "text-editor", "pdf-viewer", "external", "user-management"},
 				Options: map[string]interface{}{
-					"hideSearchBar": false,
+					"hideSearchBar":        false,
```
Comment (kulmann) [inline services/web/pkg/config/defaults/defaultconfig.go:52]: Please delete `hideSearchBar`. `false` is already the default. No need to keep it here.

File: services/web/pkg/config/defaults/defaultconfig.go [hunk 2]
```diff
@@ -49,7 +49,8 @@ func DefaultConfig() *config.Config {
...
+					"previewFileMimeTypes": []string{"image/gif", "image/png", "image/jpeg", "text/plain"},
```
Comment (kulmann) [inline services/web/pkg/config/defaults/defaultconfig.go:53]:
```suggestion
					"previewFileMimeTypes": []string{"image/gif", "image/png", "image/jpeg", "text/plain"},
```
Didn't @fschade build `tiff` support recently?

Comment (JanAckermann) [reply services/web/pkg/config/defaults/defaultconfig.go:53]: With latest master, I tried to receive previews for image/tif and image/tiff, but requests give me a 404, so I guess it's not supported yet (on latest master)

Comment (kulmann) [reply services/web/pkg/config/defaults/defaultconfig.go:53]: Then it probably was in experimental. Nevermind, thanks for checking!

Comment (kulmann) [reply services/web/pkg/config/defaults/defaultconfig.go:53]: cc @fschade maybe you need to adapt this for the experimental branch

**Review by kulmann — APPROVED** [review]

**Comment by ownclouders** [comment]
:boom: Acceptance test [<strong>Core-API-Tests-ocis-storage-3</strong>](https://drone.owncloud.com/owncloud/ocis/14979/45/1) failed. Further test are cancelled...

**Pull request merged**